sources/Context.cpp

```diff
@@ -105,6 +110,27 @@ Context::process() {
   return true;
 }
 
+void
+Context::filterDynModels() {
+  const auto& automatonsConfig = dynamicDataBaseManager_.assemblingDocument().dynamicAutomatons();
+  for (const auto& automaton : automatonsConfig) {
+    if (models_.models.count(automaton.id) == 0) {
+      continue;
+    }
+
+    const auto& modelDef = models_.models.at(automaton.id);
+    for (const auto& macroConnect : automaton.macroConnects) {
+      auto found = std::find_if(
+          modelDef.nodeConnections.begin(), modelDef.nodeConnections.end(),
+          [&macroConnect](const algo::DynModelDefinition::MacroConnection& macroConnection) { return macroConnection.id == macroConnect.macroConnection; });
+      if (found == modelDef.nodeConnections.end()) {
+        LOG(debug) << "Dynamic model " << automaton.id << " is only partially connected to network so it is removed from exported models" << LOG_ENDL;
```

Comment:
would be nice to have a warning here

Reply:
Warning means a behaviour that shouldn't happen. Erasing models that will not be exported is expected behaviour

Reply:
well an automaton that is partially connected does not seem right to me

Reply:
It is a normal behaviour on Recollements when an automaton connects two regions but the use case only deals with one region